master/internal/api_experiment.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/determined-ai/determined/master/pkg/searcher"
 	"github.com/determined-ai/determined/proto/pkg/apiv1"
 	"github.com/determined-ai/determined/proto/pkg/experimentv1"
+
```

Comment:
blocking: this will fail linting. Make sure to format the code.

Reply:
doesn't fail ci and my make fmt doesn't catch it. https://github.com/determined-ai/determined/pull/864/checks?check_run_id=916804275 what should I do?

Reply:
I can manually remove this line but is that the on instnace?

Reply:
Ya, that would be fine. I would file a ticket with the devops-ag to maybe see what is going on here.

Reply:
done. about the ci: it's not falling on my machine and it agrees with CI so I don't have the error to file the ticket. since it fails on your set up could you file that ticket?